swarm/tools/flow_studio_ui/fragments/65-context-budget-modal.html

```
@@ -3,7 +3,7 @@
      aria-modal="true" aria-labelledby="context-budget-modal-title"
      data-uiid="flow_studio.modal.context_budget">
   <div class="settings-content">
-    <button class="modal-close" aria-label="Close"
+    <button class="modal-close" title="Close modal" aria-label="Close modal"
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)